src/main/java/fr/cnes/sonar/report/utils/CommandLineManager.java

```diff
@@ -47,6 +47,7 @@
             {"s", "server", Boolean.TRUE.toString(), "Complete URL of the targeted SonarQube server."},
             {"t", "token", Boolean.TRUE.toString(), "SonarQube token of the SonarQube user who has permissions on the project."},
             {"p", "project", Boolean.TRUE.toString(), "SonarQube key of the targeted project."},
+            {"b", "branch", Boolean.TRUE.toString(), "Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin."},
```

Comment:
Again, could you please specify the default value in the description ?

Reply:
```suggestion
            {"b", "branch", Boolean.TRUE.toString(), "Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin. Default: usage of main branch."},
```

Reply:
Thanks, looks good to me.

Reply:
Could you please commit the suggested changes ?
Then I will validate the pull request.